compiler/rustc_span/src/source_map.rs

```diff
@@ -453,7 +406,9 @@ impl SourceMap {
         }
     }
 
-    pub fn span_to_string(&self, sp: Span) -> String {
+    /// Format the span location to be printed in diagnostics. Must not be included
+    /// in build artifacts as it may leak local file paths.
+    pub fn span_to_diagnostic_string(&self, sp: Span) -> String {
```

Comment:
>I checked everywhere this is used and I'm fairly confident that it's only ever used for debug/diagnositc prints. 

...and that's wrong. It seems like, at least in `rustc_mir::util::pretty` they all seem to use it for comments emitted for `InstrumentCoverage`